File: python/01/quarto.py
#!/usr/bin/env python
# -*- coding: latin1 -*-

qtd_masculina  = 0;
soma_masculino = 0;
qtd_feminia	= 0;
soma_femino = 0;


for i in range(5):
	sexo  = str(raw_input("Digite um sexo (M)asculino ou (F)eminino: "));
	idade = int(raw_input("Digite uma idade: "));

	if (sexo == 'M') or (sexo == 'm'):
		qtd_masculina += 1;
		soma_masculino = soma_masculino + idade;
	elif (sexo == 'F') or (sexo == 'f'):
		qtd_feminia += 1;
		soma_femino = soma_femino + idade;

print "Quantidade de pessoas do sexo masculino: ", qtd_masculina;
print "Quantidade de pessoas do sexo feminino: ", qtd_feminia;
print "Soma das idades do sexo masculino: ", soma_masculino;
print "Soma das idades do sexo feminino: ", soma_femino;

File: php/01/quarto.php
<?php

$sexo = array('Masculino', 'Masculino', 'Feminino', 'Masculino', 'Feminino');
$idade = array(20,25,30,12,10);

$soma_masculino = 0;
$qtde_masculino = 0;

$soma_feminino = 0;
$qtde_feminino = 0;

for($i=0;$i<5;$i++){
	echo "Sexo: ".$sexo[$i];
	echo " | Idade: ".$idade[$i]."<br />";

	if($sexo[$i] == 'Masculino'){
		$qtde_masculino++;
		$soma_masculino = $soma_masculino + $idade[$i];
	}else{
		$qtde_feminino++;
		$soma_feminino = $soma_feminino + $idade[$i];
	}
}

echo "<h1> Resultados </h1>";
echo "Total de masculino: ".$qtde_masculino."<br />";
echo "Total de feminino: ".$qtde_feminino."<br />";
echo "Idade do masculino: ".$soma_masculino."<br />";
echo "Idade do feminino: ".$soma_feminino;
?>